test/testvar.cpp

```diff
@@ -409,19 +409,19 @@ TEST(Var, productDividedByNumber)
 TEST(Var, multiplicationOfExpPosSymbol)
 {
     const Var aPos("a", Var::Sign::POSITIVE);
-    const Var expected(pow(aPos, Var(10, 3)));
-    Var pow(aPos);
+    const Var expected(tsym::pow(aPos, Var(10, 3)));
+    Var res(aPos);
 
-    pow = pow.toThe(2);
-    pow = pow.toThe(5);
-    pow = pow.toThe(Var(1, 3));
+    res = res.toThe(2);
+    res = res.toThe(5);
+    res = res.toThe(Var(1, 3));
 
-    CHECK_EQUAL(expected, pow);
+    CHECK_EQUAL(expected, res);
 }
 
 TEST(Var, noMultiplicationOfExpUnclearSymbol)
 {
-    Var res(pow(a, Var(1, 3)));
+    Var res( tsym::pow(a, Var(1, 3)));
```

Comment:
do you mind removing the space between 'res(' and 'tsym::pow' here? i know, it's pedantic :)